composer.json

```diff
@@ -1,5 +1,5 @@
 {
   "autoload": {
-    "psr-4": {"theNamespace\\": "src/"}
+    "psr-4": {"Pokedex\\": "src/"}
```

Comment:
is this namespace polymorphic across the project. I remember story 3 having a different namespace